.github/workflows/go-releaser.yml

```diff
@@ -12,28 +12,248 @@ permissions:
+            "http://timestamp.apple.com/ts01", #https://gist.github.com/Manouchehri/fd754e402d98430243455713efada710
+            "http://tsa.starfieldtech.com",
+            "http://www.startssl.com/timestamp",
+            "http://timestamp.verisign.com/scripts/timstamp.dll",
+            "http://timestamp.globalsign.com/scripts/timestamp.dll",
+            "https://rfc3161timestamp.globalsign.com/advanced"
+        )
+
+        $ex = $null
+        $signSuccessful = $false
+
+        foreach ($url in $timestampurl) {
+          Write-Host "Signing and timestamping with server $url"
+          try {
+            & AzureSignTool sign `
+              -kvu "${{ secrets.AZURE_KEYVAULT_URL }}" `
+              -kvt ${{ secrets.AZURE_KEYVAULT_TENANT_ID }} `
+              -kvi "${{ secrets.AZURE_KEYVAULT_CLIENT_ID }}" `
+              -kvs "${{ secrets.AZURE_KEYVAULT_CLIENT_SECRET }}" `
+              -kvc ${{ secrets.AZURE_KEYVAULT_CERTIFICATE_NAME }} `
+              -d "Octopus CLI" `
+              -du "https://octopus.com" `
+              -tr $url `
+              -v `
+              $env:MSI_FILE
+
+            $signSuccessful = $true
+          }
+          catch {
+            $ex = $_
+          }
+
+          if ($signSuccessful) { break }
+        }
+
+        if (-not $signSuccessful) {
+          Write-Error $ex
+          exit 1
+        }
+
+    - name: Upload MSI
+      shell: bash
+      env:
+        MSI_FILE: ${{ steps.buildmsi.outputs.msi }}
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      if: "!contains(needs.goreleaser.outputs.version, '-')" # skip prereleases
+      run: gh release upload "${{ needs.goreleaser.outputs.tag_name }}" "$MSI_FILE"
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: artifacts
+        path: ${{ steps.buildmsi.outputs.msi }}
+
+  generate-packages:
+    needs: [goreleaser, msi]
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/checkout@v3
+      with:
+        repository: OctopusDeploy/linux-package-feeds
+        token: ${{ secrets.INTEGRATIONS_FNM_BOT_TOKEN }}
+        path: linux-package-feeds-repo
+
+    - run: |
+        ln -s linux-package-feeds-repo/source linux-package-feeds
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: artifacts
+        path: artifacts/
+
+    - name: Download nuget.exe
+      run: curl -o ./nuget.exe https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
+
+    - name: Create Chocolatey package
+      env:
+        MSI_FILE: ${{ needs.msi.outputs.msi_file }}
+      run: |
+        cp artifacts/$(basename "${MSI_FILE//\\//}") build/tools
+        mono nuget.exe pack ./build/cli.nuspec -Version ${{ needs.goreleaser.outputs.version }} -OutputDirectory artifacts/
+
+    - name: Compress packages
+      run: |
+
+        zip -rj artifacts/octopus-cli.${{ needs.goreleaser.outputs.version }}.zip \
```

Comment:
given that this appears to be uploaded to the github release, perhaps we should give it a more descriptive name like `octopus-cli-allplatforms-bundled.zip` or something so that random passersby don't attempt to download it?

Reply:
This is not uploaded to the GitHub release, only as an artifact on the build and then pushed to Octopus.

Reply:
ah, right, thanks!